_includes/themes/restart/default.html

```diff
@@ -112,4 +113,5 @@
   {% endif %}
   {% include JB/analytics %}
   </body>
+{% include themes/restart/content.html %}
```

Comment:
Move this `include` inside the if statement which begins at line 109 ( I mean this if: `{% if thisnodes[0].datafile %}` )

This way the template will be added only to the "job listing" pages and not to other pages like index.html

Reply:
ok